Makefile

```diff
@@ -43,7 +43,7 @@ docs: build
 	@git diff --exit-code -- ./docs/command/
 
 mockgen: ensure-mockgen
-	./hack/mockgen.sh
+	$(GO) generate ./...
```

Comment:
Would you minding add `${BUILDFLAGS}` after `go generate`? I think this might fix the CI error.

Reply:
Done.

Since we're here, I might suggest calling your build target something without `mock` in the name, since `go generate` can be used to generate other things. But I didn't want to change that here in case you have something else relying on the current name.

Reply:
Great suggestion. I think that makes sense. Maybe I can add in a later pr.